### DISCUSSION TITLE
How to styling subRows

### DISCUSSION THREAD

**Discussion by HenryHong91** [KevinVandy/material-react-table#1112] [Q&A] [answered]
![image](https://github.com/KevinVandy/material-react-table/assets/115530962/b409e8ca-40a9-49be-a182-cc705d1e3f93)

subRows follow the parent data tree, but some of the data might be unnecessary, and yet they retain the parent table's styling. Additionally, I want to apply specific colors only to the sub-rows, but I couldn't find the correct API for this.

**Comment by KevinVandy** [comment, marked as answer]
The customize styling docs give plenty of conditional styling tips and show you how to do this. See the row instance APIs also. The tree editing example shows it too.
